docs/decisions/0014-enterprise-catalog-sync-catalogs.rst

```diff
@@ -0,0 +1,43 @@
+Catalog syncing behavior with enterprise-catalog service
+------------------------------------------------------------------------------
+
+Status
+======
+
+Accepted
+
+Context
+=======
+ECS team would like to have a way to reuse a saved set of courses configured by a query and use them in multiple catalogs across customers.
```

Comment:
tiny nit: it's more than just a specific team that wants this. The enterprise sub-system as a whole benefits from this re-usability of queries.